pkg/queue/bounded_queue_test.go

```diff
@@ -33,7 +33,7 @@ import (
 // In this test we run a queue with capacity 1 and a single consumer.
 // We want to test the overflow behavior, so we block the consumer
 // by holding a startLock before submitting items to the queue.
-func TestBoundedQueue(t *testing.T) {
+func helper(t *testing.T, startConsumers func(q *BoundedQueue, consumerFn func(item interface{}))) {
```

Comment:
hehe, I didn't mean the name `helper` literally, was just too lazy to type what you had, but that's fine.

Reply:
hahaha, I couldn't come up with anything better so I left it like you said 😄